source/renderer/app/containers/loading/SystemTimeErrorPage.js

```diff
@@ -13,25 +13,25 @@ export default class SystemTimeErrorPage extends Component<Props> {
 
   render() {
     const { stores } = this.props;
-    const {
-      localTimeDifference,
-      forceCheckTimeDifferenceRequest,
-      forceCheckLocalTimeDifference,
-      ignoreSystemTimeChecks,
-    } = stores.networkStatus;
+    // const {
+    //   localTimeDifference,
+    //   forceCheckTimeDifferenceRequest,
+    //   forceCheckLocalTimeDifference,
+    //   ignoreSystemTimeChecks,
+    // } = stores.networkStatus;
 
     const { currentLocale } = stores.profile;
 
     const { openExternalLink } = stores.app;
 
     return (
       <SystemTimeError
-        localTimeDifference={localTimeDifference}
+        localTimeDifference={0}
         currentLocale={currentLocale}
         onExternalLinkClick={openExternalLink}
-        onCheckTheTimeAgain={forceCheckLocalTimeDifference}
-        onContinueWithoutClockSyncCheck={ignoreSystemTimeChecks}
-        isCheckingSystemTime={forceCheckTimeDifferenceRequest.isExecuting}
+        onCheckTheTimeAgain={() => {}}
+        onContinueWithoutClockSyncCheck={() => {}}
+        isCheckingSystemTime={false}
```

Comment:
@disassembler do we need those props?

Reply:
@tomothespian that was my change. Yes we do... It is dirty I know :(